From 54edd20cba4ef1ead6b1b70985e046c2b6acc02f Mon Sep 17 00:00:00 2001
From: Jammy Zhou <Jammy.Zhou@amd.com>
Date: Mon, 9 Nov 2015 12:42:52 +0800
Subject: [PATCH 06/39] amdgpu: expose the AMDGPU_GEM_CREATE_NO_EVICT flag v2

With this flag specified, the buffer will be pinned at allocation time.

v2:
8991a3cc7ff9947474aa071c1dc2fd88d9808f16
[Flora Cui]
change AMDGPU_GEM_CREATE_NO_EVICT flag definition
  to avoid conflict with upstream flags.
7987cf4262d2cf83f32eb5debef9aefa791c1505
[Junwei Zhang]
update GEM flag bit for 64-bit extension

Change-Id: Iad8c81302ebf468dc45233ca7a91e6c83cb27a5f
Signed-off-by: Jammy Zhou <Jammy.Zhou@amd.com>
Reviewed-by: Chunming Zhou <david1.zhou@amd.com>
Signed-off-by: Flora Cui <Flora.Cui@amd.com>
Signed-off-by: Junwei Zhang <Jerry.Zhang@amd.com>
---
 include/drm/amdgpu_drm.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/drm/amdgpu_drm.h b/include/drm/amdgpu_drm.h
index 51b1d51..b3ce535 100644
--- a/include/drm/amdgpu_drm.h
+++ b/include/drm/amdgpu_drm.h
@@ -88,6 +88,10 @@ extern "C" {
 /* Flag that allocating the BO should use linear VRAM */
 #define AMDGPU_GEM_CREATE_VRAM_CONTIGUOUS	(1 << 5)
 
+/* Hybrid specific */
+/* Flag that the memory allocation should be pinned */
+#define AMDGPU_GEM_CREATE_NO_EVICT		(1ULL << 31)
+
 struct drm_amdgpu_gem_create_in  {
 	/** the requested memory size */
 	__u64 bo_size;
-- 
2.7.4

